parachains/integration-tests/emulated/collectives/collectives-polkadot/src/tests/fellowship.rs

```diff
@@ -16,62 +16,54 @@
 
 //! Integration tests concerning the Fellowship.
 
-use crate::*;
-use collectives_polkadot_runtime::fellowship::FellowshipSalaryPaymaster;
-use frame_support::traits::{
-	fungibles::{Create, Mutate},
-	tokens::Pay,
-};
-use sp_core::crypto::Ss58Codec;
-use xcm_emulator::TestExt;
-
 #[test]
+#[ignore]
 fn pay_salary() {
-	let asset_id: u32 = 1984;
-	let pay_from: AccountId =
-		<AccountId as Ss58Codec>::from_string("13w7NdvSR1Af8xsQTArDtZmVvjE8XhWNdL4yed3iFHrUNCnS")
-			.unwrap();
-	let pay_to = Polkadot::account_id_of(ALICE);
-	let pay_amount = 9000;
+	// let asset_id: u32 = 1984;
```

Comment:
why comment out if you put `ignore` attribute? is it not compiling?

Reply:
CI executes ignored tests